tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -20,12 +20,13 @@
 from nncf import CompressWeightsMode
 from nncf import SensitivityMetric
 from nncf.data.dataset import Dataset
+from nncf.experimental.tensor import Tensor
 from nncf.openvino.graph.node_utils import get_const_value
-from nncf.openvino.quantization.mixed_precision import MIXED_PRECISION_CRITERIA
 from nncf.quantization import compress_weights
-from nncf.quantization.algorithms.weight_compression.compression_info import WeightCompressionConfig
-from nncf.quantization.algorithms.weight_compression.quantize import get_integer_quantization_error
-from nncf.quantization.algorithms.weight_compression.quantize import reshape_weights_for_grouped_quantization
+from nncf.quantization.algorithms.weight_compression.config import WeightCompressionConfig
+from nncf.quantization.algorithms.weight_compression.mixed_precision import MIXED_PRECISION_CRITERIA
+from nncf.quantization.algorithms.weight_compression.weight_lowering import get_integer_quantization_error
```

Comment:
FYI
I made a ticket to make a template tests for weights compression algorithm to reuse references and tests No 129446

Reply:
Thanks, good suggestion!